airflow/executors/kubernetes_executor.py

```diff
@@ -483,17 +326,23 @@ def run_next(self, next_job: KubernetesJobType) -> None:
         if command[0:3] != ["airflow", "tasks", "run"]:
             raise ValueError('The command must start with ["airflow", "tasks", "run"].')
 
+        base_worker_pod = PodGenerator.deserialize_model_file(self.kube_config.pod_template_file)
+        if not base_worker_pod:
+            raise AirflowException("could not find a valid worker template yaml at {}"
+                                   .format(self.kube_config.pod_template_file))
+
         pod = PodGenerator.construct_pod(
             namespace=self.namespace,
             worker_uuid=self.worker_uuid,
             pod_id=self._create_pod_id(dag_id, task_id),
             dag_id=dag_id,
             task_id=task_id,
+            kube_image=self.kube_config.kube_image,
             try_number=try_number,
             date=execution_date,
             command=command,
-            kube_executor_config=kube_executor_config,
-            worker_config=self.worker_configuration_pod
+            kube_mutation_object=kube_executor_config,
+            base_worker_pod=base_worker_pod
```

Comment:
wherever this signature was changed, if we haven't already have it in Updating.md we should add it

Reply:
@kaxil do we need to note this if it's not an external class?